### PR TITLE
Mobile BottomBar Calc Align Menu

### PR DESCRIPTION
**Pull request by Andreas-Kainz**
Move Align settings into a drop down menu instead of buttons, cause calc has 7 align options.

Signed-off-by: Andreas-Kainz <kainz.a@gmail.com>
Change-Id: I1bbe1ed8b8ef7ada8e56486fef44b962e8dc50d7

### CODE REVIEW

File: cypress_test/integration_tests/mobile/calc/bottom_toolbar_spec.js [hunk 1]
```diff
@@ -126,54 +126,6 @@ describe('Interact with bottom toolbar.', function() {
 			.should('have.attr', 'colspan', '1024');
 	});
 
-	it('Apply left/right alignment', function() {
```
Comment (vmiklos) [inline cypress_test/integration_tests/mobile/calc/bottom_toolbar_spec.js:129]: Why did you remove this test? If it's by accident, could you please restore it? Thanks. CC @pedropintosilva

Comment (Andreas-Kainz) [reply cypress_test/integration_tests/mobile/calc/bottom_toolbar_spec.js:129]: I will get some help to write tests, so hopefully I can define that at calc, impress/draw the align stuff will be available within a submenu. But first I have to learn how to write tests.
In addition I don't know what's the benefit of this tests.
